packages/v4/src/content/design-guidelines/usage-and-behavior/pagination/pagination.md

```diff
@@ -4,42 +4,48 @@ section: usage and behavior
 hideTOC: true
 showTitle: true
 ---
-Pagination is used when you want to break up data into multiple pages. It is typically used within a table, list or card view.  If you choose to use infinite scrolling, then pagination would not be necessary. 
+Pagination splits data across multiple pages, typically within a table, list, or card view. If you use infinite scrolling, you don't need to use pagination.
+
+Add pagination above and below content views so users can easily navigate as they read through listed items on a page.
 
 ![top-bottom-pagination](./img/top-bottom-pagination.png)
 
-**1. Top pagination:** Pagination is always right-aligned in the toolbar. If pagination is required, we recommend using pagination both above and below content views.
+**1. Top pagination:** Always right-align pagination in the top toolbar, above content views.
 
-**2. Bottom pagination:** Pagination should be shown below the content view as well.
+**2. Bottom pagination:** Right-align pagination below content views, too.
 
 ## Full pagination
 
-If you do not have to worry about spatial limitations within the top toolbar, we recommend you use full pagination. By adding this to your toolbar, you will have to limit the other actions added in the toolbar on desktop and tablet views. The full variation does condense into only an item count upon mobile.
+Use full pagination unless you're restricted to a narrow workspace. Full pagination occupies more toolbar territory, so you'll have limited room for other items like bulk selectors, buttons, filters, or input fields. 
+
+[Overflow menus](http://patternfly.org/v4/design-guidelines/usage-and-behavior/toolbar#overflow-menus) allow you to incorporate these toolbar actions using less horizontal space. Toolbar actions automatically form into an overflow menu at specific breaking points. Learn how to customize these breakpoints in our [Toolbar](/design-guidelines/usage-and-behavior/toolbar) documentation.
+
+Desktop and mobile views connect with the same breakpoint specifications. On mobile, full pagination condenses to an item count. As a toolbar item, you can add and keep the item count in two ways: in both desktop and mobile views, or just in mobile view.
 
 Desktop full pagination in toolbar
 ![full-pagination](./img/full-pagination.png)
 
-**1. Items in view:** Selects the item count per page (how many total items are listed per page).
+**1. Items in view:** Allows the user to select the item count (number of listed items) per page.
 
-**2. Full pager:** Gives the user all pagination options necessary including page-back and page-forward, page-first and page-last, and a manual option to type in the page number.
+**2. Full pager:** Supplies all necessary pagination options including page-back and page-forward, page-first and page-last, and a manual option to type in a desired page number.
 
 Mobile full pagination in toolbar
 ![mobile-pagination](./img/mobile-pagination.png)
```

Comment:
This was a pre-existing error, but could you just add an extra "return" between these two lines, so that the text and image don't appear on the same line in the preview?

Should go from this:
Mobile full pagination in toolbar
![mobile-pagination](./img/mobile-pagination.png)

To this:
Mobile full pagination in toolbar

![mobile-pagination](./img/mobile-pagination.png)

Reply:
Should I add an extra "return" between these ones too?

Desktop full pagination in toolbar
![full-pagination](./img/full-pagination.png)

and 

Desktop compact pagination in toolbar
![compact-pagination](./img/compact-pagination.png)

Reply:
Sure! Probably a good idea although the pictures are bigger here so the same issue wouldn't happen but I would say it's good practice :)